src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,33 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) throws Exception {
+
+        SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+        Cipher cipher = Cipher.getInstance("AES");
+        cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+        byte[] encrypted = cipher.doFinal(value.getBytes());
+        return StringHelper.byteArrayToHexString(encrypted);
+
+    }
+
+    @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        String plaintextLength28 = "AAAAAAA";
+        //Using plaintext > 128 bits to check any differences operations.
+        String plaintextLength128 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
```

Comment:
@sujeet14108 you can use this, e.g

```java
String plaintextLength28 = generateStringOfLength(7, 'A');
```

@damithc good idea. Should this PR's scope be increased slightly so that the tests I wrote in the previous PR have this method used? Or should that be its own issue?